### PR TITLE
Update pipwin.py

### PR DESCRIPTION
**Pull request by SmartManoj**

### CODE REVIEW

File: pipwin/pipwin.py [hunk 1]
```diff
@@ -311,8 +312,7 @@ def install(self, requirement):
         Install a package
         """
         wheel_file = self.download(requirement)
-        pip._internal.main(["install", wheel_file])
-
+        subprocess.check_call([sys.executable, '-m', 'pip', 'install', 'my_package'])
```
Comment (lepisma) [inline pipwin/pipwin.py]: You need to replace `'my_package'` with `wheel_file`.

File: pipwin/pipwin.py [hunk 2]
```diff
@@ -311,8 +312,7 @@ def install(self, requirement):
         Install a package
         """
         wheel_file = self.download(requirement)
-        pip._internal.main(["install", wheel_file])
-
+        subprocess.check_call([sys.executable, '-m', 'pip', 'install',  wheel_file])
```
Comment (lepisma) [inline pipwin/pipwin.py]: not working as `sys` is not in scope. can you fix that? also can you also change the uninstall command to use subprocess as `_internal` is not working any more

**Comment by SmartManoj** [comment]
how to get notification for AppVeyor build failed

**Comment by lepisma** [comment]
not sure, i believe you can get notifications if checks fail for your pr. anyway, merging this :tada:

**Comment by SmartManoj** [comment]
> i believe you can get notifications if checks fail for your pr

nope

https://github.com/pullreminders/backlog/issues/69#issuecomment-450206220

**Pull request merged**